providers.tf

```diff
@@ -1,16 +1,19 @@
 # This is the default provider that is used to create resources inside
 # the Users account
 provider "aws" {
+  region = "us-east-1"
```

Comment:
Do we prefer this way, or do we want to define a default region variable with a default value of `us-east-1`?  It doesn't matter very much, but whatever we do here is likely to become the standard for all our Terraform modules.  We should get it the way we want it.

I personally think it makes sense to create the variable, on the off chance that someone wants to use this module in a single-account setup where all providers are really just the default provider.

Reply:
I originally had a variable in my testing branch. I switched to the non-variable way to follow how the module is used in the example. I am happy to switch to a variable with a default value if we want to go in that direction.

Reply:
When in doubt, make it a variable.  I'd rather err on the side of flexibility.

Reply:
@dav3r I felt the same way and made this change in ad358f1